bundle/edu.gemini.pot/src/test/scala/edu/gemini/spModel/gemini/seqcomp/BaselineReorderTest.scala

```diff
@@ -27,53 +27,57 @@ final class BaselineReorderTest extends GcalExecTest {
   val arc  = CalImpl(Set(Lamp.AR_ARC),            Shutter.CLOSED, Filter.NIR,   Diffuser.IR, 1, 15.0, 1, arc = true )
 
   @Test def testOrderChange(): Unit = {
+    locking {
```

Comment:
Given that this is in tests and as you say this will be soon deprecated I'm quite ok with this